src/libraries/System.Private.CoreLib/src/System/Number.NumberToFloatingPointBits.cs

```diff
@@ -113,6 +150,662 @@ public FloatingPointInfo(ushort denormalMantissaBits, ushort exponentBits, int m
             1e22,   // 10^22
         };
 
+        private static readonly ulong[] s_Pow5128Table = {
+        0xeef453d6923bd65a, 0x113faa2906a13b3f,
```

Comment:
nit: The indentation here doesn't match the indentation of the other tables.

A comment explaining what these values are would also be beneficial, IMO. Unlike `Pow10Table`, these values are "non-obvious" at first glance

Reply:
ok, I'll add a not explaining what it represents.